src/auto-complete/OptionList.tsx

```diff
@@ -87,6 +87,17 @@ const OptionsList = forwardRef<OptionsListRef, OptionsListProps>((props: Options
   // 键盘事件，上下选择
   const onKeyInnerPress = (e: KeyboardEvent) => {
     if (e.code === 'Enter' || e.key === 'Enter') {
+      // 当没有匹配项的时候， 默认回车选中第一个
+      const currentIndex = activeIndexRef.current;
+
+      if (currentIndex === -1) {
+        // currentIndex = 0
+
+        return onSelect?.('', { e });
```

Comment:
```suggestion
        return
```

这里或许不应该 return 一个不存在的选项值出来，用户还需要去处理这种特殊情况，直接 return 终止逻辑会更好一些